src/vivarium_csu_alzheimers/components/testing.py

```diff
@@ -25,25 +25,29 @@
 class Testing(Component):
     """Marks simulants as having been tested if they meet the eligibility criteria."""
 
+    @property
+    def time_step_priority(self) -> int:
+        return 6
+
     @property
     def columns_created(self) -> list[str]:
         return [
             COLUMNS.TESTING_PROPENSITY,
-            COLUMNS.TESTED_STATUS,
+            COLUMNS.TESTING_STATE,
             COLUMNS.BBBM_TEST_DATE,
             COLUMNS.BBBM_TEST_RESULT,
         ]
 
     @property
     def columns_required(self) -> list[str]:
         return [
-            ALZHEIMERS_DISEASE_MODEL.MODEL_NAME,
+            COLUMNS.DISEASE_STATE,
             COLUMNS.AGE,
```

Comment:
Nit: could remove comma to get this all on one line. There are a few other instances of this in this file.